package/src/frontend/browser/browser.js

```diff
@@ -100,32 +104,35 @@ export const browser = (function(){
           }
         };
 
-        promises[name] = promises[name] || $.when(runTest(name));
+        promises[name] = promises[name] || Promise.all([runTest(name)]).then((a) => {return a[0]});
         return promises[name];
       };
 
       asyncHas.not = function(name) {
-        return asyncHas(name).pipe(function(result) {
+        return asyncHas(name).then(function(result) {
           return !result;
         });
       };
 
       asyncHas.all = function(/* arguments */) {
-        return $.when.apply(null, arguments).pipe(function(/* arguments */) {
-          return _.all(arguments);
+        return Promise.all(arguments).then(function(results) {
+          return _.all(results);
         });
       };
 
-      $.when.apply(null, _.map(_.keys(tests), function(name) {
+      Promise.all(_.map(_.keys(tests), function(name) {
         return asyncHas(name).then(function(result) {
           var cssClassName = name.replace(/ /g, '_');
 
-          $('body').toggleClass('has_' + cssClassName, !!result);
-          $('body').toggleClass('has_no_' + cssClassName, !result);
+          document.body.classList.toggle('has_' + cssClassName, !!result)
+          document.body.classList.toggle('has_no_' + cssClassName, !result)
```

Comment:
So far `pageflow-paged/frontend` did not require polyfills. Now we need at least one for `Promise` and `classList` for IE11 (https://caniuse.com/#search=classlist). For `Promise` we can use the `core-js` polyfill that we already use in scrolled.

Reply:
added `core-js` and `classlist-polyfill` to `pageflow-paged` package